tests/installation/test_executor.py

```diff
@@ -846,38 +1007,6 @@ def test_executor_should_write_pep610_url_references_for_git_with_subdirectories
     )
 
 
-def test_executor_should_use_cached_link_and_hash(
```

Comment:
This test is replaced by the more complete `test_executor_should_write_pep610_url_references_for_non_wheel_urls`